cypress/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "target": "es5",
+        "lib": ["dom", "dom.iterable", "esnext"],
+        "allowJs": true,
+        "skipLibCheck": true,
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true,
+        "strict": false,
+        "forceConsistentCasingInFileNames": true,
+        "noFallthroughCasesInSwitch": true,
+        "module": "esnext",
+        "moduleResolution": "node",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "noEmit": true,
+        "jsx": "react-jsx"
+    },
+    "include": ["src", "**/*.ts"]
```

Comment:
This doesn't look right to me. Firstly I think it'd be better to have this file at root level, call it `tsconfig.cypress.json`, and let it extend the main config file. And, more importantly, we should make sure we use it to compile the correct files, which obviously isn't `src`.

Reply:
I had several issues with using TS and running this in Docker which is why the configuration file isn't at root level. That being said, `src` can probably be removed.